charybdis/insn.py

```diff
@@ -23,6 +23,7 @@ class InsnName(enum.Enum):
     JP = "JP"
     JR = "JR"
     LD = "LD"
+    LDH = "LDH"
```

Comment:
I was going to argue against including this, but it makes things much simpler. Fine... 😢